examples/vhdl/external_buffer/run.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The example contains no comment or documentation that explains what is the goal and what is being demonstrated by the example.

Reply:
Indeed, should this be considered an example or an internal test?

Reply:
An automated test of functionality provided by VUnit is always necessary regardless. The question is if the test can also serve as an example. To serve as an example it must also be pedagogical. However a test should also be documented to some extent. I am fine with just making this an internal test for now.

Reply:
In the end, I left the two external testbenches here, and added comments so it is also pedagogical.